src/transformers/models/altclip/processing_altclip.py

```diff
@@ -16,10 +16,23 @@
 Image/Text processor class for AltCLIP
 """
 
-import warnings
+import sys
+from typing import List, Union
 
-from ...processing_utils import ProcessorMixin
-from ...tokenization_utils_base import BatchEncoding
+
+if sys.version_info >= (3, 11):
+    from typing import Unpack
+else:
+    from typing_extensions import Unpack
```

Comment:
Similar to what we have with `PILImageResampling` could we define an object, where we do this importing once in one place and then import from there?

Reply:
ah that's a good point! will do that